code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -249,6 +252,25 @@
 
 	visible_message(span_danger("[src] curls up into a ball!"))
 
+	// chest maybe because getting slammed in the chest would knock it off your face while dead
+	AddComponent(/datum/component/knockoff, knockoff_chance = 40, target_zones = list(BODY_ZONE_HEAD, BODY_ZONE_CHEST), slots_knockoffable = slot_flags)
+
+/obj/item/clothing/mask/facehugger/allow_attack_hand_drop(mob/living/carbon/human/user)
```

Comment:
Also because I just realized facehugger toys are considered alive: 
```suggestion
/obj/item/clothing/mask/facehugger/allow_attack_hand_drop(mob/living/carbon/human/user)
	if(!real || sterile)
		return ..()
```